tests/integration/components/lt-head-test.js

```diff
@@ -92,6 +94,34 @@ test('render sort icons', function(assert) {
   assert.notOk(hasClass(sortIcon, 'fa-sort-asc'));
 });
 
+test('custom iconComponent has arguments', function(assert) {
+  const iconSortable = 'unfold_more';
```

Comment:
`assert.expect(6)`
otherwise the test will pass if the component doesn't get rendered

Reply:
@alexander-alvarez Will add this soon, thanks alexander I'm learning quite a bit from this PR